src/transcript/TranscriptEvent.ts

```diff
@@ -95,6 +96,14 @@ export class TranscriptEventConverter {
                 transcriptItem.vocabularyFilterMatch = item.vocabularyFilterMatch;
               }
 
+              if (item.stable) {
+                transcriptItem.stable = item.stable;
+              }
+
+              if (item.confidence) {
```

Comment:
Not sure what this string `item.confidence` can be. But if `item.confidence` is an empty string, `transcriptItem.confidence` remains `undefined`.

In JavaScript:

```
if ('') {
   // Not executed
}
```